build_test.go

```diff
@@ -544,17 +544,17 @@ func TestBuildInto(t *testing.T) {
 		t.Run(fixture.name, func(t *testing.T) {
 			t.Parallel()
 			fixture := fixture
-			sc := new(StringsCreator)
+			creator := new(StringsCreator)
```

Comment:
un-needed changes, unless it was flagged?

Reply:
This, and most of the PR's changes, seem to be fixing complaints from the `varnamelen` linter.  `varnamelen` seems too strict to me; what are both your thoughts on disabling it?

Reply:
Yeah, the `varnamelen` linter complained about the variable names, if you guys are fine with it, I can disable it.

Reply:
i'll be ok with that, reading https://github.com/blizzy78/varnamelen, seems like it's pretty strict. We can leave it for another day.